src/sudoers/mod.rs

```diff
@@ -158,6 +159,13 @@ impl Sudoers {
                     if can_execute(path) {
                         return Some(path.to_owned());
                     }
+                    let path = resolve_path(
+                        path,
+                        &std::env::var("PATH").unwrap_or(env!("DEFAULT_PATH").to_string()),
+                    );
+                    if let Some(path) = path {
```

Comment:
One question: shouldn't we again check for `can_execute` after the path resolution and return `None` otherwise? (That might involve a small refactor on this code so we don't repeat ourselves)

Reply:
I don't know if that is really necessary because the resolve_path function also tests to see if the file it found is executable.

Reply:
Right, thanks! In that case we can merge this as-is. Thanks for the contribution!